Projects/static/Projects/js/profile.js

```diff
@@ -80,7 +80,7 @@ $(document).ready(function() {
 
   $(".delete").click(function(e) {
     e.preventDefault();
-    var pr = $(this).closest(".ind_pr");
+    var pr = $(this).closest(".pp");
     var pr_id = $(".pr_info", pr).attr("pr-id");
     var csrf = $(".ppr").attr("csrf");
```

Comment:
This won't work, as there is no class named ppr with issues on profile page, though it will call the remove_pr function yet it'll fail as no csrf will be available. 
var csrf = $(".ppr").attr("csrf");
csrf will be undefined here.

Reply:
it could be fixed by just one change and that is changing ind_pr to ppr, rest is okay :smile: